src/textual/fuzzy.py

```diff
@@ -40,6 +40,10 @@ def __init__(
             ".*?".join(f"({escape(character)})" for character in query),
             flags=0 if case_sensitive else IGNORECASE,
         )
+        self._first_word_regex = compile(
```

Comment:
Is a new `Matcher` created on each key press in the command palette？

Reply:
Yeah